common/core/web/input-processor/src/text/prediction/languageProcessor.ts

```diff
@@ -193,7 +170,7 @@ namespace com.keyman.text.prediction {
         return null;
       }
 
-      let context = new TranscriptionContext(Mock.from(target), this.configuration);
+      let context = new ContextWindow(Mock.from(target), this.configuration);
```

Comment:
Does this need a new reference at the top of the file?

Reply:
Definitely one of the "quirks" of the whole "`<reference path="" />`" set up.  Since a file that references this makes the _new_ reference first... no, not in a literal sense.

Not saying it's a bad idea, though - adding it may indeed be more clear.